.travis.yml

```diff
@@ -8,5 +8,7 @@ install:
 script: bash -ex .travis-docker.sh
 env:
   global:
+    - PINS="xapi-rrdd:. xapi-rrd-transport:. xapi-rrd-transport-utils:. rrddump:. rrd-transport:."
+  jobs:
     - PACKAGE="xapi-rrdd"
```

Comment:
How about rrd-transport? I think we want to test that one too, I think it needs a PACKAGE line